README.md

```diff
@@ -135,7 +138,6 @@ Before running e2e tests, ensure you've run `yarn install` to download dependenc
 3. Start a test build with live changes: `yarn start:test` is particularly useful for development. It starts a test build that automatically recompiles application code upon changes. This option is ideal for iterative testing and development. This command also allows you to generate test builds for various types, including:
    - `yarn start:test` for main build
    - `yarn start:test:flask` for flask build
-   - `yarn start:test:mv2` for mv2 build
```

Comment:
This is still relevant; we're only removing the need for the `ENABLE_MV3` flag for tests here. Not for builds